packages/commerce-sdk-react/src/hooks/ShopperCustomers/cache.ts

```diff
@@ -134,9 +135,37 @@ export const cacheUpdateMatrix: CacheUpdateMatrix<Client> = {
     },
     updateCustomerPassword: noop,
     updateCustomerProductList: TODO('updateCustomerProductList'),
-    updateCustomerProductListItem(customerId, {parameters}) {
+    updateCustomerProductListItem(customerId, {parameters}, response) {
```

Comment:
What do you think of this cache-update logic? 

The selective update proves to be tricky because the property to update is nested deeply in the data. For example, Typescript complained about the possibility of some things being `undefined`. So you'll see that conditions/guards are added in this code block.